frontend/src/store/modules/setting.ts

```diff
@@ -18,6 +19,16 @@ export const useSettingStore = defineStore("setting", {
   state: (): SettingState => ({
     settingByName: new Map(),
   }),
+  getters: {
+    workspaceSetting(state): WorkspaceGeneralSettingPayload | undefined {
+      const setting = state.settingByName.get("bb.workspace.general");
+      if (!setting || !setting.value) {
+        return;
+      }
+
+      return JSON.parse(setting.value) as WorkspaceGeneralSettingPayload;
```

Comment:
we can use `WorkspaceGeneralSettingPayload.fromJSON(JSON.parse(setting.value));`. So that we do not need the `as WorkspaceGeneralSettingPayload`.